gui/wxpython/wxplot/histogram.py

```diff
@@ -99,7 +99,7 @@ def OnCreateHist(self, event):
         """
         try:
             self.SetCursor(StockCursor(wx.CURSOR_ARROW))
```

Comment:
I think in this case I would remove the try except block completely, we use setCursor elsewhere without it.